src/components/jellyfinActions.ts

```diff
@@ -506,12 +526,12 @@ export function getDownloadSpeed(byteSize: number): Promise<number> {
  */
 export function detectBitrate(): Promise<number> {
```

Comment:
For this function my PR changes to await syntax. So no changes necessary just in this instance.